rigs/heft-web-rig/profiles/library/config/heft.json

```diff
@@ -46,6 +46,22 @@
        * An optional object that provides additional settings that may be defined by the plugin.
        */
       // "options": { }
+    },
+    {
+      /**
+       * The path to the plugin package.
+       */
+      "plugin": "@rushstack/heft-jest-plugin",
+
+      /**
+       * An optional object that provides additional settings that may be defined by the plugin.
+       */
+      "options": {
+        /**
+         * If set to true, Jest will pass a test run when no tests are discovered.
+         */
+        "passWithNoTests": true
```

Comment:
This option actually isn't a documented Jest configuration option, and it can only be passed via CLI. Passing via config will cause Jest to warn, which will cause the build to fail. Unsure how it got to this point without being an issue to be honest.

Reply:
What if a rig has `heft-jest-plugin` set up with this set to `false`, but a project wants to enable this option? Would the project not be able to use the rig?

Seems like that scenario means that this isn't the right place for this option.

Reply:
The only other place that I can think of to put this would be in the test stage properties so it can be passed in on the command line. But then that would need to be passed in to `heft test` in every package.json since we default to `false` for boolean values... that's the only way that makes sense to me. I'd really want to avoid tweaking the `jest.config.json` any further than adding the `extends` property. Whatever is in that file (other than `extends` since it gets stripped out during loading) should be 100% compatible with Jest imo since we're loading it in and passing straight to Jest.

Reply:
>I'd really want to avoid tweaking the `jest.config.json` any further than adding the `extends` property. 

Agreed.

@octogonz - what do you think?

Reply:
It actually seems pretty reasonable as a CLI arg. It makes it explicit when a project has no tests, and should (ideally) only be required occasionally. For example, it was only needed in the Webpack plugins for this PR, which also points out a lack of tests in those plugins.